app/client/templates/task/components/TaskTimeSlotsCompontent.html

```diff
@@ -7,227 +7,274 @@
-                        </div>
-                    {{/if}}
-                </div>
-            {{/if}}
-        {{/unless}}
+                        {{/if}}
+                    </div>
+                {{/if}}
+            {{/unless}}
+        </div>
+
     </div>
 </template>
 
 <template name="updateTimeSlotCalendar">
-    <div class="creneau {{state}}" data-timeslotdid="{{this._id}}" style="height: {{height}};">
-
-        <div class="wrapper on-calendar">
-
-            {{#each getPeopleNeededMerged _id}}
-                <!--<div class="peopleNeed" style="height: {{../height}};">-->
-                <!--<div class="">-->
-
-                <div class="col-md-1 people-need">
-                    {{#unless userId}}{{count}}{{/unless}}
-                    {{#if userId }}
-                        <div class="user">
-                            <i class="mdi mdi-account mdi-inline"></i>
-                        </div>{{/if}}
-                    {{#if teamId}}
-                        <div class="team">
-                            <i class="mdi mdi-account-multiple mdi-inline"></i>
-                        </div>{{/if}}
-                        <!--{{#ifNotEmpty skills}}-->
-                    <div>
-                        {{#each skills}}
-                            <div class="skill">
-                                <i class="mdi mdi-android-studio mdi-inline"></i>
-                            </div>
-                        {{/each}}
-                    </div>
-                        <!--{{/ifNotEmpty}}-->
-                </div>
-                <!--</div>-->
-            {{/each}}
-
-            <div class="col-md-1 people-need">
-                <div>{{getAlreadyAssignedPeopleNeedCount _id}}</div>
-                <div>people need already assigned</div>
+    <div class="creneau {{state}} {{#if equals currentTimeSlot._id this._id}}currentTimeSlot{{/if}}" data-timeslotdid="{{this._id}}" style="height: {{height}};">
+        <div class="on-calendar">
+
+            <div class="col-xs-2 no-padding creneau-userId" title="
+                {{getUserIdNeedCount _id}} particular {{#if isMore (getUserIdNeedCount _id) 1}}people are{{else}}person is{{/if}} needed">
+                <p class="no-padding creneau-header">
+                <div>{{getUserIdNeedCount _id}}</div>
+                </p>
+                <p class="col-xs-12 no-padding people-need">
+                    <i class="mdi mdi-account"></i>
+                </p>
             </div>
 
+            <div class="col-xs-8 no-padding creneau-to-affect">
+                {{#each getPeopleNeededMergedWithoutUserId _id}}
+                    {{#unless userId}}
+                        <p class="col-xs-2 no-padding people-need" title="
+                            {{count}} need{{#unless equals count 1}}s{{/unless}} of :{{#if teamId}} a particular team{{#if skills}}, {{/if}}{{/if}}{{#each skills}}{{#unless equals @index 0}}and {{/unless}} {{skillLabel}} (skill) {{/each}}">
```

Comment:
(trop de paddings, le string de title doit rentrer sur une ligne de code)

Ca fait trop d'infos pour une tooltip qui n'est pas vraiment faite pour afficher plus d'une ligne de texte. 

Que penses tu de : 
3 needs of 3 skills
3 needs of a team
3 needs of a team and 3 skills